.github/workflows/build.yml

```diff
@@ -242,6 +249,24 @@ jobs:
         name: ${{ matrix.os }} build
         path: ppsspp/
 
+    - name: Create release
+      if: startsWith(github.ref, 'refs/tags/') && runner.os == 'macOS' && matrix.extra == 'test'
+      run: |
+        cd ppsspp || exit 1
+        mv PPSSPPSDL.zip PPSSPPSDL-${GITHUB_REF##*/}.zip || exit 1
+
+    - name: Upload release
+      uses: softprops/action-gh-release@v1
+      if: startsWith(github.ref, 'refs/tags/') && runner.os == 'macOS' && matrix.extra == 'test'
+      with:
+        files: ppsspp/*.zip
+        body: >
```

Comment:
Well, it feels _natural_ to me to edit the release message afterwards with e.g. a changelog manually. This code is just a convenience message to have _something_ before this happens. There is no restriction to the amount of edits, so I feel there is no big issue in leaving it as is, especially since there is no good place to source the release message automatically anyway. Correct me if you think I am wrong.

Reply:
Well, I don't think anyone will edit them, but okay.  Honestly, I'd prefer the release notes were just the version again like now:
https://github.com/hrydgard/ppsspp/releases

Rather than a message that repeats for every single release.  Those are not "release notes", those are a description (this repo already has one of those.)

-[Unknown]